DataFormats/Scouting/src/classes_def.xml

```diff
@@ -39,14 +39,14 @@
   <class name="Run3ScoutingVertex" ClassVersion="3">
       <version ClassVersion="3" checksum="316446070"/>
   </class>
-  <class name="Run3ScoutingElectron" ClassVersion="3">
-      <version ClassVersion="3" checksum="1086011373"/>
+  <class name="Run3ScoutingElectron" ClassVersion="4">
+      <version ClassVersion="4" checksum="1250202632"/>
   </class>
   <class name="Run3ScoutingMuon" ClassVersion="3">
       <version ClassVersion="3" checksum="3882497397"/>
   </class>
-  <class name="Run3ScoutingPhoton" ClassVersion="3">
-      <version ClassVersion="3" checksum="1683146807"/>
+  <class name="Run3ScoutingPhoton" ClassVersion="4">
+      <version ClassVersion="4" checksum="1725280278"/>
```

Comment:
Add the new class version, but keep the original one as well.

Reply:
It would be 
```xml
<class name="Run3ScoutingPhoton" ClassVersion="4">
  <version ClassVersion="3" checksum="1683146807"/>
  <version ClassVersion="4" checksum="1725280278"/>
```
```

Reply:
That PR shows an example of how the xml file was updated (increasing the class version, yes, but also keeping the checksums of the previous versions), i.e.
```xml
  <class name="Run3ScoutingParticle" ClassVersion="4">
      <version ClassVersion="3" checksum="1672617274"/>
      <version ClassVersion="4" checksum="1541586682"/>
  </class>
```
I'm not an expert, but `scram` will basically provide the updated xml for you. I would suggest to read
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning
https://github.com/cms-sw/cmssw/pull/28349/files#r343383848

Reply:
(Thanks, @Dr15Jones , I saw your comments only now.)